DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -22,7 +22,9 @@ public class RegistrationPromptsController : Controller
         public const string AddPromptAction = "addPrompt";
         public const string NextAction = "next";
         public const string SaveAction = "save";
+        public const string BulkAction = "bulk";
         private const string CookieName = "AddRegistrationPromptData";

```

Comment:
Should this now be "AddPromptCookieName" (and the other one might need "Prompt" in the name too) :)